src/index.tsx

```diff
@@ -47,13 +46,21 @@ if (window.location.hostname !== 'localhost') {
           // i.e no 429s
           [430, 599],
         ],
+
+        failedRequestTargets: [/^(?!.*\.?(alchemy|snapshot)\.(com|org)).*$/],
```

Comment:
As we go, we may want to go with a *whitelist* request instead to keep things clean, but that's probably fine for now